ui/src/components/app-card/app-card.css

```diff
@@ -21,7 +25,7 @@
   height: 130px;
   padding: 33px 0;
   overflow: hidden;
-  background: linear-gradient(90deg, #fff 0%, #ece9e6 95.83%);
+  /* background: linear-gradient(90deg, #fff 0%, #ece9e6 95.83%); */
```

Comment:
Can we remove this?

Reply:
I put it back in, I was testing if it was needed, it's a fall back background gradient so leaving as was.